pkg/structure/hash.go

```diff
@@ -208,15 +208,22 @@ func (t *TxStructure) HGetLastN(key []byte, num int) ([]HashPair, error) {
 
 // HClear removes the hash value of the key.
 func (t *TxStructure) HClear(key []byte) error {
+	var keys []kv.Key
```

Comment:
will this part accumulate and take many memory, say several GBs?

Reply:
IDK how much memory will this part take.

If the keys take several GBs memory, the underlying MemBuffer will also be that size after the delete.

And this is the only correct way to do this operation, delete during an iteration is a UB.